content/docs/searching.md

```diff
@@ -139,30 +140,30 @@ The **Filters** list is a space-separated list of wildcard expressions that cmd.
 * You can also exclude certain file patterns by prefixing the filter with a `!`;
 for example, **Filters:  `!*.bin *.*`** will exclude files matching `*.bin` from the search results, but include any other filename.
 * As of Notepad++ v8.2, you can also exclude particular folders from the search: The exclusion operator is always `!` at the beginning of the expression, so in order to distinguish folder from file, `\` should be used as prefix of the folder name/pattern, following `!`. That allows the exclusion of the directories under the root directory you want to search (the 1st level of matched directories).
-If users need to exclude folders with the same name (or names matched the specific pattern) in all levels, the `+` should be put between `!` and `\` to exclude them recursively. For example:
-    * `!\tests` will not search any files in the `tests` folder
-    * `!\bin*` will not search any files in the `bin` folder or `bin64` folder (or any other directory that matches `bin*`)
-    * `!+\log*` will _recursively_ not search any files in folders that start with log (so directories like `.\log`, `.\logs`, `.\other\logfiles`, `.\many\layers\deep\log` will all be excluded from the search)
+If users need to exclude folders with the same name (or names matching the specific pattern) in all levels, the `+` should be put between `!` and `\` to exclude them recursively. For example:
+    * `!\tests` will not search any files in the `tests` folder,
+    * `!\bin*` will not search any files in the `bin` folder or `bin64` folder (or any other directory that matches `bin*`),
+    * `!+\log*` will _recursively_ not search any files in folders that start with log (so directories like `.\log`, `.\logs`, `.\other\logfiles`, `.\many\layers\deep\log` will all be excluded from the search).
 
-    *Note*: "inclusion of folder" is not allowed, and such pattern will be ignored
+    *Note*: Inclusion of subfolders of excluded folders is not allowed, and any such pattern will be ignored.
 
-* As of Notepad++ v8.2, if you hover your mouse cursor over the **Filters:** label, a helpful popup will show example syntax for you
+* As of Notepad++ v8.2, if you hover your mouse cursor over the **Filters** label, a helpful popup will show example syntax to you.
 * Please also note that the PathMatchSpec() Windows API is being used for the **Filters**, as its behavior departs from cmd.exe wildcard parsing sometimes.
-* To find only files that have no extension, you cannot just say `*.`, because of quirks in the PathMatchSpec().  Instead, you can search for any file that _doesn't_ have at least one letter in the extension: `!*.?*`
+* To find only files that have no extension, you cannot just say `*.`, because this would still include a `.`.  Instead, you can search for any file that _doesn't_ have at least one letter in the extension: `!*.?*`.
```

Comment:
> this would still include a `.`.

That's not technically the reason.  Try this experiment at the command line: create a file named `a` with no extension, then do `dir *.` -- you will notice that it finds `a` even though it doesn't have the dot.  

Maybe a better phrasing would be:
> you cannot just say `*.` (despite that filter working in a Windows command prompt)

Or just not give a reason-clause, if you think that any attempted clarification will confuse readers more than it clarifies.

Reply:
This `.` goes to you.